R/correctionsataglance-render.R

```diff
@@ -116,14 +116,28 @@ plotLanes <- function(gsplotObject, laneData, whichCol,
              labels = laneData[[labelName]], cex=1) 
     }
     
-    if(any(names(laneData) %in% 'numText')){  
+    if(any(names(laneData) %in% 'numText')){   
       i <- which(!is.na(laneData$numText))
+      if (any(laneData$moveText)) { 
+        pos <- NA
+        dateRange <- format(addData$dateData$dateRange, "%m/%d/%Y")
+        xyText <- format(laneData$xyText$x, "%m/%d/%Y")
```

Comment:
can you rename this so it is more obvious that it is the x position of the xyText? I thought it was that actual text